visualization/package.json

```diff
@@ -118,13 +125,17 @@
 		"shelljs": "^0.8.4",
 		"three": "^0.126.1",
 		"three-orbit-controls": "^82.1.0",
-		"typescript-json-schema": "^0.50.0"
+		"typescript-json-schema": "^0.50.0",
+		"zone.js": "^0.11.4"
```

Comment:
is this required?

Reply:
yes, zone.js is a peer dependency, which needs to be loaded before Angular.